canvas_modules/common-canvas/__tests__/notification-panel/notification-panel-test.js

```diff
@@ -332,19 +328,35 @@ describe("toolbar notification icon state renders correctly", () => {
 			canvasController
 		);
 
-		expect(wrapper.find(".notification-panel")).to.have.length(1);
+		expect(container.querySelectorAll(".notification-panel")).to.have.length(1);
 		expect(canvasController.getNotificationMessages().length).to.equal(0);
-		expect(wrapper.find(".notification-panel-empty-message-container")).to.have.length(1);
+		expect(container.querySelectorAll(".notification-panel-empty-message-container")).to.have.length(1);
 
 		canvasController.setNotificationMessages([notificationMessage0]);
-		wrapper.update();
-		expect(wrapper.find(".toggleNotificationPanel-action")).to.have.length(1);
-		expect(wrapper.find(".notification-panel-empty-message-container")).to.have.length(0);
+		createIntlCommonCanvasRTLRerender(
```

Comment:
Why is another renderer needed here? Is there any `wrapper.update()` equivalent in RTL?

Reply:
Sure, I will try to replace it with act().

Reply:
@srikant-ch5 Couple of issues with this code - 

1. When I try to run just this test by adding `only`, it fails. 
2. `createIntlCommonCanvasRTLRerender()` method is calling `<CommonCanvas / >` again. Earlier test wasn't doing this but it was waiting for `canvasController.setNotificationMessages([notificationMessage0]);` this config to take effect before checking the next code. I think using `async await` here will make the code much cleaner. 
Found this sample code here - https://medium.com/@AbbasPlusPlus/react-testing-library-understanding-act-and-when-to-use-it-301bd06fd1bc
```
import { render } from '@testing-library/react';

it('should fetch and display data', async () => {
  const { findByText } = render(<AsyncComponent />);
  
  const message = await findByText('Hello');
  expect(message).toBeInTheDocument();
});
```